NumberBaseball/NumberBaseball/main.swift

```diff
@@ -6,5 +6,59 @@
 
 import Foundation
 
-print("Hello, World!")
 
+var numbersByComputer: [Int] = []
+var numberOfChance: Int = 9
+var numbersByUser: [Int] = []
+
+func returnRandomNumbers() -> [Int] {
+    var numberSet = Set<Int>()
+    
+    while(numberSet.count < 3) {
+        numberSet.insert(Int.random(in:1...9))
+    }
+
+    return Array(numberSet)
+}
+
+func returnResult() -> String {
+    var strikeCount: Int = 0
+    var ballCount: Int = 0
+    
+    for i in 0...2 {
+        if numbersByUser[i] == numbersByComputer[i] {
+            strikeCount += 1
+        } else if numbersByComputer.contains(numbersByUser[i]) {
+            ballCount += 1
+        }
+    }
+    return "\(strikeCount) 스트라이크, \(ballCount) 볼"
+}
+
+func startGame() {
+    numbersByComputer = returnRandomNumbers()
+    
+    while(numberOfChance > 0) {
+        numbersByUser = returnRandomNumbers()
+        print("임의의 수 : ", terminator: "")
+        numbersByUser.forEach {
+            print($0, terminator: " ")
+        }
+        print()
+        
+        let gameResult = returnResult()
+        print(gameResult)
+        if gameResult == "3 스트라이크, 0 볼" {
+            print("사용자 승리!")
+            break
+        }
+        
+        numberOfChance -= 1
+        print("남은 기회 : \(numberOfChance)")
+        if numberOfChance == 0 {
```

Comment:
```suggestion
        if numberOfChance == .zero {
```

`0` 이라는 표현 대신 Int 타입이 들어오는 곳이 확실하다면 `.zero` 라는 표현을 쓸 수도 있다는 팁 공유드립니다~!
굳이 바꾸실 필요는 없어요! 😄